src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -5,6 +5,7 @@
 #include "SettingContainer.h"
 #include "SettingContainer.g.cpp"
 #include "LibraryResources.h"
+#include "../TerminalSettingsModel/LegacyProfileGeneratorNamespaces.h"

```

Comment:
is this required? Rather not include things directly out of TSM

Reply:
Ah, this was a relic of when we were detecting the specific profile generator. Removed.

Reply:
Had to add this back in to be able to detect when we're overriding a setting from the proto-extension vs one of our profile generators